app/views/coronavirus_landing_page/components/landing_page/_statistics_section.html.erb

```diff
@@ -7,58 +7,42 @@
   } %>
 
   <% if statistics && statistics.cumulative_vaccinations %>
-    <div>
-      <%= render "govuk_publishing_components/components/heading", {
-        text: "Total vaccinations"
-      } %>
-
-      <span class="govuk-heading-xl">
-        <%= number_with_precision(statistics.cumulative_vaccinations / 1_000_000.0, precision: 1) %>
-        million
-      </span>
-
-      <p class="govuk-body">people have received their first dose</p>
-
-      <p class="govuk-body">
-        Last updated <%= statistics.cumulative_vaccinations_date.strftime("%d %B %Y") %>
-      </p>
-    </div>
+    <%= render "coronavirus_landing_page/components/landing_page/statistic_section_block", {
+      title: "Total vaccinations",
```

Comment:
Not something we need to resolve in this PR but we should determine whether this copy belongs in the YAML file or can be hardcoded here.

To be honest it seems simpler to me to have it hardcoded here but I don't want to create an inconsistency with everything else.

@leenagupte do you have any thoughts on this?

Reply:
Have we decided on this yet? or should we always just use the YAML?

Reply:
Leena left a comment in https://trello.com/c/WVZPCn3t/203-spike-explore-embedding-covid-statistics-into-coronavirus-page#comment-60586d99cf0973569ea001d5 to suggest YAML would be consistent 😭 

I feel like we shouldn't do the YAML until we get the final go ahead since it involves a change to another repo.